modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -15,7 +15,24 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. *Optional*: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
+Devfile::
+If your Git repository contains a Devfile it would be automatically detected.
+* The Devfile would be automatically detected and populated on the *Devfile path* field.
+* To edit the file import type, click *Edit import strategy* option, to select a different strategy.
+* If multiple Devfiles are detected, the Devfile path would need to be specified, relative to the *Context Dir* field, to import a specific Devfile.
+
+Dockerfile::
+If your Git repository contains a Dockerfile it would be automatically detected.
+* The Dockerfile would be automatically detected and populated on the *Dockerfile path* field.
+* To edit the file import type, click *Edit import strategy* option, to select a different strategy.
+* If multiple Dockerfiles are detected, the Dockerfile path would need to be specified, relative to the *Context Dir* field, to import a specific Dockerfile.
```

Comment:
```suggestion
* If multiple Dockerfiles are detected, to import a specific Dockerfile, specify the path of the Dockerfile relative to the *Context Dir* field.
```
I recommend reorganizing this sentence logically and sequentially into the following three parts:
`<precondition - "if multiple...">, <user goal - "to import...">, <user action - "specify the path...">`

It is also clearer to use the imperative (specify) instead of the subjunctive mood and passive voice (would need to be specified) mood.

The IBM SG mentions  this here:

```
Mood
Use the appropriate mood for the information type:
- Use the imperative mood for requests or instructions, such as in procedures.
 - Before you install the product, back up your files.
 - Select one or more check boxes.
- Use the indicative mood to specify information, such as facts and explanations.
 - The product provides a powerful way to store information.
 - The computer cancels the action and stops all processing.
- Do not use the subjunctive mood in technical documentation.
 - **X** If you were to save the file ...
 - **Y** If you save the file ...
 - **X** It is important that the file be saved ...
 - **Y** Important: Save the file ...
```